backend/src/openbeheer/types/selectielijst.py

```diff
@@ -1,18 +1,16 @@
 # generated by datamodel-codegen:
 #   filename:  https://selectielijst.openzaak.nl/api/v1/schema/openapi.json
-#   timestamp: 2025-08-15T10:54:09+00:00
+#   timestamp: 2025-09-29T15:28:09+00:00
 #   version:   0.30.1
 
 from __future__ import annotations
 
+from datetime import timedelta
 from enum import Enum
```

Comment:
The change above is (probably) intentional, and is the result of regenerating types. Earlier I had to add this change to make Resultaat work